helpers/freedom_helper.js

```diff
@@ -22,12 +22,12 @@ module.exports = {
     get_oauth_access_token
 };
 
-function get_oauth_access_token(params) {
+async function get_oauth_access_token(params) {
     params.client_id = client_id;
     params.client_secret = client_secret;
 
-    return accounts.generate_token(DASHBOARD_SCOPES.USER_READONLY)
-        .then(({access_token}) => {
+    return await accounts.generate_token(DASHBOARD_SCOPES.USER_READONLY)
+        .then(async ({access_token}) => {
```

Comment:
We are using async-await syntax. Why do we need `.then` here?

Reply:
@jonelrempis Aah got it, will refactor it now.

Reply:
@jonelrempis Refactored the entire code